inputs/values/ref_panel_1kg.json

```diff
@@ -315,6 +315,7 @@
         "gs://gatk-sv-ref-panel-1kg/outputs/GATKSVPipelineBatch/38c65ca4-2a07-4805-86b6-214696075fef/call-GatherSampleEvidenceBatch/GatherSampleEvidenceBatch/bb5e70f1-7186-4e29-96f8-dcb9ee000642/call-GatherSampleEvidence/shard-154/GatherSampleEvidence/f5159902-35ad-4e8e-ac76-2e6278ac387f/call-PESRCollection/PESRCollection/4538b819-c0fe-4d69-bcc1-1658a293e6de/call-RunPESRCollection/cacheCopy/NA21122.disc.txt.gz.tbi",
         "gs://gatk-sv-ref-panel-1kg/outputs/GATKSVPipelineBatch/38c65ca4-2a07-4805-86b6-214696075fef/call-GatherSampleEvidenceBatch/GatherSampleEvidenceBatch/bb5e70f1-7186-4e29-96f8-dcb9ee000642/call-GatherSampleEvidence/shard-155/GatherSampleEvidence/9dfac5cf-9a0b-4070-ad5b-2fe087a41856/call-PESRCollection/PESRCollection/066417e3-3848-4dc7-b46e-173fdf2f9ae7/call-RunPESRCollection/cacheCopy/NA21133.disc.txt.gz.tbi"
     ],
+    "PE_files_list": "gs://gatk-sv-ref-panel-1kg/outputs/GATKSVPipelineBatch/38c65ca4-2a07-4805-86b6-214696075fef/PE_files_list.txt",
```

Comment:
Seems like we will need to eliminate the array inputs to avoid redundancy / conflicts with the file lists. Can you file a ticket for that?

Reply:
I'm not sure we want to replace Array[File] inputs with File inputs for pipeline intermediates because in Terra, those intermediate arrays get written into the data table and used that way, and requiring them to be in a File would mess up that flow. We could consider providing a File? and Array[File]? option every time, but that seems cumbersome. I thought it made more sense to limit the scope to pipeline _inputs_, even though it means some redundancy